index.html

```diff
@@ -1401,9 +1401,47 @@ <h2>Core Vocabulary Definitions</h2>
 <p>
      <span class="rfc2119-assertion" id="td-context-default-language">
      In the <code>@context</code> array, the optional default language that is 
-     used throughout a TD document is defined within an object as a member with 
+     used throughout a TD document for human-readable strings outside of a <a href="#multilanguage">MultiLanguage</a> object
+     is also defined within an object as a member with 
      a key <code>@language</code> whose value MUST be a string representing a [[!BCP47]] 
```

Comment:
Should be "the name `@language` whose ... language tag."

Reply:
Talking about "member" would make it a serialization assertion.

Reply:
I'll see what I can do to clean this up.

Reply:
Done.